homeassistant/components/homee/strings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "config": {
+    "flow_title": "Homee {name} ({host})",
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "step": {
+      "user": {
+        "title": "Configure homee",
+        "description": "Manually enter the ip address of the homee you want to connect.",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "password": "[%key:common::config_flow::data::password%]",
+          "username": "[%key:common::config_flow::data::username%]"
+        },
+        "data_description": {
+          "host": "The IP address ofyour Homee.",
+          "username": "The username for your Homee.",
+          "password": "The password for your user."
```

Comment:
username for Homee and password for my user? That sounsd wierd

Reply:
I could write Homee-user on both? Or just user...